tests/test_good_sounds.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+from mirdata.datasets import good_sounds
+from tests.test_utils import run_track_tests
+
+
+def test_track():
```

Comment:
could you add a test with an example track and its expected attributes?

Reply:
well, it is checked on the JAMS but I add it!

Reply:
done